support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -217,13 +242,11 @@ class CreateSubscriptionController(
       result: EitherT[Future, CreateSubscriptionError, StatusResponse],
       product: ProductType,
       userEmail: String,
-  )(implicit request: Request[CreateSupportWorkersRequest], writeable: Writeable[String]): Future[Result] = {
+  )(implicit request: CreateRequest, writeable: Writeable[String]): Future[Result] = {
```

Comment:
I know you didn't add it, but I'm not sure why request is implicit, and I prefer to avoid using them unless it saves a lot of boilerplate.  Not sure if it's something for this PR though, it's certainly not a show stopper!

Reply:
Yea, happy to relook at the clarity of the code in this file after if we want.